actors.md

```diff
@@ -324,13 +324,35 @@ func SlashConsensusFault(block1, block2 BlockHeader) {
 		Fatal("blocks do not prove a slashable offense")
 	}
 
+	if !self.Miners.Has(msg.From) {
+		Fatal("slash collateral must only be called by a miner actor")
+	}
+
 	miner := AuthorOf(block1)
 
-	// TODO: Some of the slashed collateral should be paid to the slasher
+	// Check if miner has been slashed
+	if !self.Miners.Has(miner) {
+		Fatal("miner has been slashed")
+	}
 
 	// Burn all of the miners collateral
-	miner.BurnCollateral()
-
+	// const slashedCollateral = self.GetCollateral(miner)
```

Comment:
remove commented code

Reply:
will do after @anorth confirms that miner.Balance is the right way to get it value and we can directly set miner.Balance.

Reply:
`self.Balance` should be fine for accessing it, but in order to remove funds, they need to be moved somewhere. If burning funds, use `BurnFunds`